tools/functional-tester/etcd-tester/main.go

```diff
@@ -49,6 +49,7 @@ func main() {
 	consistencyCheck := flag.Bool("consistency-check", true, "true to check consistency (revision, hash)")
 	isV2Only := flag.Bool("v2-only", false, "'true' to run V2 only tester.")
 	stresserType := flag.String("stresser", "default", "specify stresser (\"default\" or \"nop\").")
+	failureTypes := flag.String("failures", "default,failpoints", "specify failures (concat of \"default\" and \"failpoints\").")
```

Comment:
It's not really a failpoint dependency, it queries the failpoint endpoints via http and if it can't find anything issues a warning. Should we make it fatal if failpoints is given but the failpoint endpoints aren't available?